src/main/java/com/jalasoft/jfc/model/ImageParam.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not disclose such Confidential
+ *  Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+package com.jalasoft.jfc.model;
+
+/**
+ * This class has the common attributes, Setters, and Getters of an
+ * image.
+ *
+ * @version 0.1 11 Dec 2019.
+ * @author Oscar Lopez.
+ * */
+public class ImageParam {
+    //Variable for obtain input path of Image.
+    private String inputPathFile;
+    //Variable of output path of Image.
+    private String outputPathFile;
+    //Variable of output path of Thumbnail of a Image.
+    private String outputPathThumbnail;
+    //Variable for change the width of Image.
+    private int widthOfFile;
+    //Variable for change the height of Image.
+    private int heightOfFile;
+    //Variable for change the percentage of white and black of a Image.
+    private int whiteBlankPercentage;
+    //Variable for rotate a Image.
+    private double degreesToRotate;
```

Comment:
Here you can add the variable documentation at right side of each line only intil to 80 caracters per line